ui/assets/css/_print.css

```diff
@@ -0,0 +1,38 @@
+@media print {
+
+  .usa-disclaimer, .navbar-title, .req .metadata, .req .req-text {
+    color: #111 !important;
+  }
+
+  .sidebar,
+  .tab-container {
+    display: none;
+  }
+
+  .main {
+    border-left: 0;
+  }
+
+  ol.list-reset {
+    border-bottom: 1px solid #111;
+  }
+
+  .active-filter {
+    margin: 0;
+  }
+  .requirement-list > li,
+  .policy {
+    border: 0;
+    border-bottom: 1px solid;
+    border-radius: 0;
+  }
+
+  .topics-list a, a:link {
+   text-decoration: none;
+   color: #111;
+  }
+
+  .topics-list a:link:after {
+    content: " (" attr(href) ") ";
```

Comment:
Let's remove this for now. If we have more links in the future we'll consider it again.